codes/classical/bits/hadamard.yml

```diff
@@ -0,0 +1,16 @@
+code_id: hadamard
```

Comment:
Spreadsheet says "simplex" is dual of Hamming. According to wikipedia, this is dual to extended Hamming..? Check Concise Encyclopedia of Coding Theory Def. 1.10.2 and Table 12.1.